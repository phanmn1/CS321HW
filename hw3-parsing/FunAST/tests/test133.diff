1c1
< Invalid syntax at (8,7), (
---
> Invalid syntax at (3,7), fact
