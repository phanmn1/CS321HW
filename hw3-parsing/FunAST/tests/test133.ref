Invalid syntax at (8,7), (
