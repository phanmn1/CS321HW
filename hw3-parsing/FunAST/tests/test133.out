Invalid syntax at (3,7), fact
